python/nano/src/bigdl/nano/pytorch/lightning.py

```diff
@@ -48,6 +52,10 @@ def _forward(self, batch):
         nargs = self.model.forward.__code__.co_argcount
         return self(*(batch[:nargs - 1]))
 
+    def on_train_start(self) -> None:
```

Comment:
Why this change? This PR should focus on OpenCV support; you may submit a separate PR for this change if needed.